src/node_errors.cc

```diff
@@ -1044,6 +1050,304 @@ void TriggerUncaughtException(Isolate* isolate, const v8::TryCatch& try_catch) {
 
 }  // namespace errors
 
+namespace {
+// This is annoying, but v8 does not give a nice way of getting a good
+// stack property so we have to create an error object and capture a
+// stack that way.
+Global<Value> MakeErrorStack(
```

Comment:
It doesn't seem nice that we have to re-implement all these error properties in C++. If the only problem with implementing them in JS (which seems nicer in comparison since we can do multiple inheritance there) is getting hold of C++ stuff before `internalBinding` and `require` become available, I think we can simply put JSTransferable and the symbols to some object (`primordials` would do if we are being lazy and not want to invent a new argument) in `InitializePrimordials` and pass them into the per-context script for DOMException to use?